flybywiresim/aircraft
# fix(a32nx/fcu): baro unit auto init

## Summary of Changes

Remove fs9gps from the FCU baro unit auto init, and fix it so it works (hopefully).

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Set the baro unit to auto in the flypad.
- Spawn at an airport in the USA and ensure the unit auto-inits to in.Hg.
- Spawn in most of the rest of the world and ensure the unit auto-inits to hPa.
- Set the baro unit to hPa in the flypad, spawn in the USA, and ensure the unit auto-inits to hPa.
- Set the baro unit to in.Hg in the flypad, spawn somewhere living in the future, and ensure the unit auto-inits to in.Hg.


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
